### PR TITLE
fixed description in api

### PR DESCRIPTION
**Pull request by JuliaGalabut**

### CODE REVIEW

**Review by BorisTkachenko — APPROVED** [review]

**Review by alebastrov — DISMISSED** [review]

File: oap-ws-account-api/src/main/java/oap/ws/account/UserData.java [hunk 1]
```diff
@@ -78,14 +78,18 @@ public Map<String, String> getRoles() {
 
     @JsonIgnore
     @Override
-    public String getDefaultOrganization() {
-        return user.defaultOrganization;
+    public Optional<String> getDefaultOrganization() {
+        return Optional.ofNullable( user.defaultOrganization ).or( Optional::empty );
```
Comment (alebastrov) [inline oap-ws-account-api/src/main/java/oap/ws/account/UserData.java:82]: do not need .or( Optional::empty )

File: oap-ws-account-api/src/main/java/oap/ws/account/UserData.java [hunk 2]
```diff
@@ -78,14 +78,18 @@ public Map<String, String> getRoles() {
...
     @Override
-    public String getDefaultAccount() {
-        return user.defaultAccount;
+    public Map<String, String> getDefaultAccounts() {
+        return user.defaultAccounts;
+    }
+
+    @JsonIgnore
+    public Optional<String> getDefaultAccount( String organizationId ) {
+        return Optional.ofNullable( user.defaultAccounts.get( organizationId ) ).or( Optional::empty );
```
Comment (alebastrov) [inline oap-ws-account-api/src/main/java/oap/ws/account/UserData.java:92]: same

**Review by alebastrov — CHANGES_REQUESTED** [review]

File: oap-ws-account-api/src/main/java/oap/ws/account/UserData.java (hunk 1 above)
Comment (JuliaGalabut) [reply oap-ws-account-api/src/main/java/oap/ws/account/UserData.java]: +

File: oap-ws-account-api/src/main/java/oap/ws/account/UserData.java (hunk 2 above)
Comment (JuliaGalabut) [reply oap-ws-account-api/src/main/java/oap/ws/account/UserData.java]: removed

File: oap-ws-account-api/src/main/java/oap/ws/account/UserData.java [hunk 3]
```diff
@@ -100,8 +104,8 @@ public boolean canAccessAccount( String organizationId, String accountId ) {
     }
 
     public UserData addAccount( String organizationId, String accountId ) {
-        if( user.defaultAccount == null ) {
-            user.defaultAccount = accountId;
+        if( !user.defaultAccounts.containsKey( organizationId ) ) {
```
[the remark this answers is not in the document]
Comment (JuliaGalabut) [reply oap-ws-account-api/src/main/java/oap/ws/account/UserData.java]: changed

**Review by alebastrov — DISMISSED** [review]

**Review by alebastrov — APPROVED** [review]

**Pull request merged**